delfin/exporter/base_exporter.py

```diff
@@ -25,6 +24,21 @@
 
 LOG = log.getLogger(__name__)
 
+prom_grp = cfg.OptGroup('PROMETHEUS_EXPORTER')
+kafka_grp = cfg.OptGroup('KAFKA_EXPORTER')
+
+prometheus_opts = [
+    cfg.BoolOpt('enable', default=False,
+                help='Prometheus exporter is enabled or not'),
+
+]
+
+kafka_opts = [
+    cfg.BoolOpt('enable', default=False,
+                help='Kafka exporter is enabled or not'),
+
+]
+
```

Comment:
plz move kafka opt and prometheus opt to their exporter

Reply:
Done,Thanks